src/modules/tiers/reducer.ts

```diff
@@ -0,0 +1,80 @@
+import { loadingReducer } from 'decentraland-dapps/dist/modules/loading/reducer'
+import {
+  BUY_THIRD_PARTY_ITEM_TIERS_FAILURE,
+  BUY_THIRD_PARTY_ITEM_TIERS_REQUEST,
+  BUY_THIRD_PARTY_ITEM_TIERS_SUCCESS,
+  FETCH_THIRD_PARTY_ITEM_TIERS_FAILURE,
+  FETCH_THIRD_PARTY_ITEM_TIERS_REQUEST,
+  FETCH_THIRD_PARTY_ITEM_TIERS_SUCCESS
+} from './actions'
+import {
+  TiersState,
+  BuyThirdPartyItemTiersFailureAction,
+  BuyThirdPartyItemTiersRequestAction,
+  BuyThirdPartyItemTiersSuccessAction,
+  FetchThirdPartyItemTiersFailureAction,
+  FetchThirdPartyItemTiersRequestAction,
+  FetchThirdPartyItemTiersSuccessAction
+} from './types'
+
+const INITIAL_STATE: TiersState = {
+  data: {
+    thirdParty: []
```

Comment:
Just making sure, this property is here so we can, in the future, add tiers to other parts of the state right?

I ask mostly because it's called `thirdParty` but not `thirdPartyItem`s when the type is `ThirdPartyItemTier`

Reply:
It is set like that for the purpose you've mentioned.
You're right, it should be called `thirdPartyItems` as it defines the tiers of the items in a more proper way.